src/product.html

```diff
@@ -62,18 +62,18 @@
           </section>
         </header> <!-- Risky Business Ends here -->
         <section class="col col-2-3"> <!-- Left Column Starts Here -->
-          <section class="img-slider" ng-app="slider-module"> <!-- Air Drummers Starts here -->
+          <section class="img-slider"> <!-- Air Drummers Starts here -->
             <div class="img-main">
               <a href="#" id="prev"><div class = "arrow-case"></div><i class="fa fa-arrow-left"></i></a>
-              <img class= "img-active" src={{listingImages.results[0].url_570xN}}>
+              <img class= "img-active" ng-src={{listingImages.results[0].url_570xN}}>
```

Comment:
Better to ask the API to include a `MainImage` for the `Listing` but picking the first image in the array is a good second choice...